libs/wingii/src/lib.rs

```diff
@@ -180,7 +181,7 @@ pub mod type_system {
 		pub fn load_dep(&mut self, dep: &str, search_start: &str, opts: &AssemblyLoadOptions) -> Result<AssemblyName> {
 			let is_root = opts.root;
 			let module_dir = package_json::find_dependency_directory(dep, search_start)
-				.expect(format!("could not find module {} in {}", dep, search_start).as_str());
+				.ok_or::<Box<dyn Error>>(format!("could not find module {} in {}", dep, search_start).into())?;
```

Comment:
You should be able to just get rid of the `into`, then you don't need to annotate `ok_or`. Also suggestion for the message:

```suggestion
			.ok_or(format!(
				"Unable to load \"{}\": Module not found in \"{}\"",
				dep, search_start
			))?;
```